xbmc/music/MusicDatabase.cpp

```diff
@@ -176,8 +176,6 @@ void CMusicDatabase::CreateTables()
   CLog::Log(LOGINFO, "create role table");
   m_pDS->exec("CREATE TABLE role (idRole integer primary key, strRole text)");
   m_pDS->exec("INSERT INTO role(idRole, strRole) VALUES (1, 'Artist')");   //Default role
-  m_pDS->exec("INSERT INTO role(idRole, strRole) VALUES (2, 'Composer')");
```

Comment:
I think this is fine. We are in alpha, the existence of an unused role in the databases of those that are testing the nightlies is not going to cause any harm. And yes the first library clean will remove them.